### DISCUSSION TITLE
Android Tablet Splash Screen

### DISCUSSION THREAD

**Discussion by cmd8** [expo/expo#23099] [Archived: Questions] [answered]
Hey all,

How can I add a splash screen to an Android tablet? Adding one for the iPad is pretty simple, as per the app.config docs. 
If I have to use the bare workflow for that, how do I add a separate tablet splash screen image to the ejected Android project?

Cheers

**Comment by RuslanBratus** [comment, marked as answer]
Maybe that can help u (API 31+)
https://developer.android.com/develop/ui/views/launch/splash-screen
For different splash screens (mobile/tablet) u can just have different themes file in proper directory (default/sw600dp)
